portal/src/main/webapp/js/src/load-frontend.js

```diff
@@ -39,14 +39,17 @@ window.loadReactApp = function(config) {
     if (window.localdev || window.localdist || localStorage.heroku || localStorage.netlify) {
         showFrontendPopup(window.frontendConfig.frontendUrl);
     }
+    document.write('<script src="' + window.frontendConfig.frontendUrl + 'reactapp/common.bundle.js?'+ window.frontendConfig.appVersion +'"></scr' + 'ipt>');
+    document.write('<script src="' + window.frontendConfig.frontendUrl + 'reactapp/main.app.js?'+ window.frontendConfig.appVersion +'"></scr' + 'ipt>');
+
+};
+
+window.loadAppStyles = function() {
```

Comment:
where is this `loadAppStyles` called?

Reply:
this should be called somewhere in the `head`, right?

Reply:
nice catch guys.  make the edit to a file in target directory. updated.